lib/engine/step/g_1856/cash_crisis.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
I'm re-using 1817's CashCrisis because EMR-style fundraising can be done for reasons beyond buying trains:
It can be done because:
1: Corporation needs a train
2: Corporation cannot pay interest
3: Corporation is forced to payoff loans (outside of nationalization) and cannot cover them

Reply:
It might be a better idea to modularize the re-usable chunks between the two implementations rather than extending off a different game where the implementation could be changed without an expected inheritance.

Reply:
Yeah, the 1856 cash crisis is, code-wise, based off of the core EmergencyMoney module, not on anything in 1817. I re-used the structure of the 1817 CashCrisis to be able to use the same UI elements to make it work.

Right now only 1856 and 1817(+ clones) use cash crisis so I'm personally leaning towards waiting for a 3rd game to come around that wants a cash crisis type step to make a base CashCrisis for all 3